source/auth_n.rst

```diff
@@ -4492,10 +4468,10 @@ This example covers challenging Factors that have already been created. To see a
 Challenging an SMS Factor
 """""""""""""""""""""""""
 
-To challenge an SMS Factor, you send a request like this, with or without specifying a message.
-
 .. only:: rest
 
+  To challenge an SMS Factor, you send a request like this, with or without specifying a message.
```

Comment:
I see many indentation changes which not only make quite a noise in this PR (difficult to know which changes are needed to be reviewed and which aren't) but also could cause some indentation issues in the resulting HTML. Can you please make sure that only the intended changes are committed to this PR? Thanks